scripts/GenerateSdkPackages/make-all.ps1

```diff
@@ -1,78 +0,0 @@
-[CmdletBinding(PositionalBinding=$false)]
```

Comment:
note that I don't think we ever used this directly in our build, it was a manual script to create nuget packages for the listed dlls since previously no one else created nuget packages for them (they were just shipped in VS).

However I found my email from back in 2020 where we asked for them to be published to real feeds, but not sure if that was completed (at the time there was a blocker on the vssdk side).  The blocker is now resolved though - https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1157868

It does look like the [public vs-impl feed](https://dev.azure.com/azure-public/vside/_artifacts/feed/vs-impl
) is getting updated with versions of `Microsoft.VisualStudio.Progression.CodeSchema`, but the others have versions last been published in 2020.   @mgoertz-msft do you happen to know if you are now able to publish the rest of them?  We don't need updates now, just confirmation that we won't need to manually publish these in the future (if we require updates)

Reply:
@dibarbet  The reason `Microsoft.VisualStudio.Progression.CodeSchema` is being published is because it was added as a WrapperPackage in the VS repo for the purpose of creating that package automatically. However, if they are not used by your build why would we ever need to publish these?

Reply:
@mgoertz-msft we use the packages in our build, but not this script :)
E.g. https://github.com/dotnet/roslyn/blob/main/eng/Versions.props#L170

Reply:
Yeah, this script was never ran as a part of the build; it was a manual thing you'd run when you needed to make packages. The paths point to outputs in the VS legacy drop, so it's not like a Roslyn build could even consume them.

In the unlikely case we need to update to newer packages here and we don't actually have new internal packages to share directly, we could always revert this. Or, just fix the VS build to actually make the packages we wanted originally. I'd like to think we have much better tooling there now than we did years ago when this was needed.